starcounter-include.html

```diff
@@ -310,27 +310,30 @@
                 return this._renderCompositionChange(compositionProviderComposition, () => {
                     // convert string from view-model, to actual document fragment
                     const customComposition = this.stringToDocumentFragment(compositionProviderComposition);
-
+                    
                     // append default compositions for views not covered by this composition
-                    if (compositionProvider.ViewUris) {
-                        const keys = Object.keys(this.viewModel);
-                        for(let key of keys) {
-                            const scoped = this.viewModel[key];
-                            if (scoped && scoped.Html && compositionProvider.ViewUris.indexOf(scoped.Html) === -1) {
-                                if (this.defaultComposition) {
-                                    this.template.scopedNodes.forEach((nodes) => {
-                                        if(nodes.scope === key) {
-                                            nodes.forEach((node) => appendComposition(customComposition, node));
-                                        }
-                                    });
-                                }
-                                else {
-                                    // BUG,TODO (tomalec): looks like a bug
-                                    // This means that if there *is* a custom composition, but it does not cover all views,
-                                    // and it happend that none of the views had a default composition,
-                                    // we would discard custom composition and use default -> fallback
-                                    return false;
-                                }
+                    const defaultViewHtmlsToBeRendered =
+                        compositionProvider.ActualViewUris.filter(
+                                x => !compositionProvider.ViewUris.includes(x.hasOwnProperty("BlendingUri") ? x.BlendingUri : x))
```

Comment:
Can you add a test for the path that contain `BlendingUri`?

Reply:
Today is @PatrykSzwer's last day, so he wont add anything but I may. 

> Can you add a test for the path that contain `BlendingUri`?

This whole section never had a single test. This PR does not make it much worse, but I agree tests should be added. https://github.com/Starcounter/starcounter-include/blob/e7a9b1ef3429d8851e3857ec98568f35813f0101/starcounter-include.html#L315-L339